src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2471,6 +2471,13 @@ well as menu structures (for main menu and popup menus).
         windowMode="main"
         desc="Clear console"/>
 
+   <cmd id="buildClear"
+        buttonLabel=""
+        label="Clear Build"
+        menuLabel="Clear Build"
```

Comment:
Since this command isn't on a menu you don't need a menuLabel. If this is ever added to the main menu it won't build because there is no access key ("_"). So, suggest just removing menuLabel.